sdk/storage/azure-storage-blob-nio/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.0.0-beta.9 (Unreleased)
-
+## 12.0.0-beta.9 (2021-09-15)
+### Dependency Updates
```

Comment:
Replace this with:
### Other Changes
#### Dependency Updates
- Updated blob dependency to 12.14.0